luigi/parameter.py

```diff
@@ -661,3 +661,28 @@ def serialize(self, cls):
         Converts the :py:class:`luigi.task.Task` (sub) class to its family name.
         """
         return cls.task_family
+
+
+class EnumParameter(Parameter):
+    """
+    A parameter whose value is an :class:`~enum.Enum`.
+
+    .. code:: python
+
+        my_param = EnumParameter(enum=MyEnum)
```

Comment:
Can you just copy in the documentation you added in the PR description (how it integrates with CLI) into here as well? TaskParameter already has similar documentation you can copy paste from if you're unsure about rst syntax.